scripts/run_pylint_check.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This scripts runs the pylint checker on a given input directory
+
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+
+usage() { echo "Usage: $0 -i <pylint input directory> -o <pylint output directory> [-e <comma seperated excluded directories>]" 1>&2; exit 1; }
+
+while getopts ":i:o:e:" p; do
+    case "${p}" in
+        o)
+            o=${OPTARG}
+            ;;
+        i)
+            i=${OPTARG}
+            ;;
+        e)
+            e=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${i}" ] || [ -z "${o}" ]; then
+usage
+fi
+
+if [ -z "${e}" ]; then IGNORE=""; else IGNORE="--ignore=${e}"; fi
+
+echo "Running pylint with:"
+echo "Input directory: ${i}"
+echo "Output directory: ${o}"
+echo "Ignoring directories: ${e}"
+pylint ${IGNORE} --output-format=parseable --reports=y  ${i} > ${o}/pylint.out 2> ${o}/pylint.log
```

Comment:
Better to use gumby-exported $OUTPUT_DIR var for the output, defaulting to PWD.
Remember to add some documentation with the correct comment syntax so it is found by the config generator. (https://github.com/Tribler/gumby/blob/devel/scripts/generate_config_file.py#L46)

In general it's better to use config options than command arguments as it will be easier to discover by other users.